openpype/hosts/webpublisher/plugins/publish/collect_published_files.py

```diff
@@ -142,7 +148,8 @@ def _process_sequence(self, files, task_dir):
             "name": ext[1:],
             "ext": ext[1:],
             "files": files,
-            "stagingDir": task_dir
+            "stagingDir": task_dir,
+            "tags": ["review"]
```

Comment:
I wonder if we should modify settings a little so we will be able to assign tags per extension so for example here https://github.com/pypeclub/OpenPype/pull/1876/files#diff-ab754e471e01c2dedc81d21db9d578dd1d02a89c7e22975bf3b0c22b40f9f72dR16 we would be having:

```
{
	"render": {
        "is_sequence": true,
        "extensions": {
			"tvp": {"tags": []},
            "png": {"tags": ["review", "burnin"]},
            "exr": {"tags": ["review", "generate-slate"]}
        },
        "families": [
            "review"
        ],
        "subset_template_name": ""
    }
}
```

Reply:
or better 
```{
	"render": {
        "is_sequence": true,
        "extensions": [
			"tvp",
            "png",
            "exr"
        ],
        "families": [
            "review"
        ],
		"tags": [
			"review", 
			"generate-slate"
		],
        "subset_template_name": ""
    }
}```

Reply:
Implemented.